src/context/AuthContext.js

```diff
@@ -0,0 +1,59 @@
+import { createContext, useState } from "react";
+import jwt_decode from "jwt-decode";
+import { APIUrl } from "../config/config";
+
+export const AuthContext = createContext({});
+
+export default function AuthContextProvider({ children }) {
+  const [username, setUsername] = useState("abczzz13");
+  const [password, setPassword] = useState("v$*UfieAepz7zuYphJ^^");
```

Comment:
It's not smart to put credentials like this in open source code. 

I could wipe out the production database with this (possibly) 

So _never_ put _any_ credentials in your source like this.